prog.go

```diff
@@ -163,8 +163,12 @@ func newProgramWithBTF(spec *ProgramSpec, btf *btf.Handle, opts ProgramOptions)
 		_, logErr = bpfProgLoad(attr)
 	}
 
+	if errors.Is(logErr, unix.EPERM) && logBuf[0] == 0 {
```

Comment:
Could we add `only emit the error mentioning rlimit if the verifier buffer is empty` as a comment here for posterity?

Reply:
Done!